composer.json

```diff
@@ -14,15 +14,14 @@
     },
     "require": {
         "php": ">=5.5",
-        "zendframework/zend-crypt": "~2.5",
+        "zendframework/zend-crypt": "dev-develop as 2.6.0",
```

Comment:
If we're removing zend-servicemanager as a dep, the change is essentially backwards compatible.

Couldn't this dependency be `^2.5 || ^3.0`? That would make it possible to keep this component in the 2.X series.